notebooks/pytorch/training.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
nit: Change filename to: Training with PyTorch.ipynb ?

Reply:
Done